addons/mail/static/src/messaging/service/messaging_service/messaging_service.js

```diff
@@ -75,6 +102,105 @@ const MessagingService = AbstractService.extend({
     getEnv() {
         return this.env;
     },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @returns {Object}
+     */
+    _generateEntities() {
```

Comment:
Remove this private method, and use/patch something in `env` instead of entity for `startLoopFetchImStatus`.

Or maybe just `Messaging(Initializer)` `start` should take a parameter to know if the loop has to be called or not?

Whatever the solution, this private method should not exist, and no new patch should be introduced in tests except on `env`.